soh/soh/SohMenuBar.cpp

```diff
@@ -201,8 +201,7 @@ void DrawSettingsMenu() {
 
             static std::unordered_map<LUS::AudioBackend, const char*> audioBackendNames = {
                 { LUS::AudioBackend::WASAPI, "Windows Audio Session API" },
-                { LUS::AudioBackend::PULSE, "PulseAudio" },
-                { LUS::AudioBackend::SDL, "SDL" },
+                { LUS::AudioBackend::SDL, "SDL" }
```

Comment:
this is because https://github.com/Kenix3/libultraship/pull/421 is on latest LUS main